products/compute/api.yaml

```diff
@@ -14662,7 +14662,6 @@ objects:
     collection_url_key: 'items'
     input: true
     has_self_link: true
-    min_version: beta
     description: |
       Represents a VPN gateway running in GCP. This virtual device is managed
       by Google, but used only by you. This type of VPN Gateway allows for the creation
```

Comment:
Should the `api` reference on line 14674 be updated to from `https://cloud.google.com/compute/docs/reference/rest/beta/vpnGateways` to `https://cloud.google.com/compute/docs/reference/rest/v1/vpnGateways`?

Reply:
Oh, that the link to the docs. I'll fix that.